com.unity.multiplayer.transport.utp/Runtime/UTPTransport.cs

```diff
@@ -58,16 +49,23 @@ private enum State
 
         public override ulong ServerClientId => m_ServerClientId;
 
-        public string RelayJoinCode => m_RelayJoinCode;
-
         public ProtocolType Protocol => m_ProtocolType;
 
+        private RelayServerData m_RelayServerData;
+
+        private static readonly RelayServerData k_DefaultRelayServerData = default(RelayServerData);
+        private static RelayServerData DefaultRelayServerData => k_DefaultRelayServerData;
```

Comment:
Any reason why this is not just a private constant?

Reply:
Or why we're not using `default(RelayServerData)` directly when we need it?

Reply:
> Any reason why this is not just a private constant?

As far as the lambda operator, nothing I can rightly recall, honestly.

> Or why we're not using `default(RelayServerData)` directly when we need it?

My initial thought was that instantiating a new instance of a default, well-known RelayServerData constant for each comparison check would be less optimal than simply having a constant defined, but I have not verified that the C# compiler doesn't simply optimize the "non-optimal" version anyway, nor would it be terribly expensive anyhow, and adds more code complexity for relatively little gain, since this doesn't occur in a performance-critical loop.

I have no problem changing both of these points.